dev/cookbooks/dev/files/default/user_default.erl

```diff
@@ -6,6 +6,7 @@
 -module(user_default).
 % required to expose the record definitions we load in from various includes
 -compile(export_all).
+-compile(nowarn_export_all).
```

Comment:
It might be useful to mention the reason this needs to be added. export_all is usually not a good practice. and we do that only in tests. But if not an issue to track and fix it where needed would also be of help.

Reply:
Added comment in source file.